Prereq ImageMagick missing in documentation (or shouldn't be  a hard prereq)

- OS version: macOS 12.3.1
 - Compiler version: AppleClang 13.1.6.13160021
 - DD4hep version: v01-20-02
 - Reproduced by: 
On a mac without ImageMagick, ```cmake -Bbuild -DUSE_EXTERNAL_CATCH2=OFF```
 - Input: n/a
 - Output: I don't think the full log is helpful.
 Final three lines before errors:
```
-- Found LaTeX compiler.
-- Found UnixCommands: /bin/bash
-- I could not find the pdftops command.
```
Afterward, infinite avalanche of errors:
```
CMake Error at cmake/LATEX.cmake:1137 (message):
  Could not find convert program.  Please download ImageMagick from
  http://www.imagemagick.org and install.
Call Stack (most recent call first):
  cmake/LATEX.cmake:1209 (latex_add_convert_command)
  cmake/LATEX.cmake:1244 (latex_convert_image)
  cmake/LATEX.cmake:1497 (latex_process_images)
  cmake/LATEX.cmake:1892 (add_latex_targets_internal)
  doc/usermanuals/DD4hep/CMakeLists.txt:2 (ADD_LATEX_DOCUMENT)
```
Stopped by ctrl-c

 - Goal:
 This was easy enough to find a solution for with homebrew, but a) endless errors are not graceful, and b) is ImageMagick really needed?

--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -38,6 +38,8 @@ IF(LATEX_COMPILER AND BIBER_COMPILER)
     ADD_SUBDIRECTORY(usermanuals/DDEve)
     ADD_SUBDIRECTORY(usermanuals/DDG4)
     ADD_SUBDIRECTORY(usermanuals/DDRec)
+
+    MESSAGE(STATUS "Tried to set up Latex, if you saw errors, please disable building the documentation (BUILD_DOCS=OFF) or install missing packages.")
     
 ELSE()
     MESSAGE(STATUS "No LaTeX/Biber found, cannot compile user manual.")

